pkg/builtins/builtins.go

```diff
@@ -436,6 +436,7 @@ func Init() {
 	})
 }
 
-func docs(u string, extra ...string) []string {
-	return append(extra, fmt.Sprintf("url=%s", u))
+func docs(string, ...string) []string {
+	// TODO(sr): bring this back when we've moved the builtin docs to the OPA website
```

Comment:
Interesting. I guess let's put something in our calendars so we follow up on this TODO in a week or two? Hopefully the docs will be moved over by then.